util/include/cryptopp_digest_creator.hpp

```diff
@@ -31,15 +31,13 @@ class CryptoppDigestCreator : public DigestCreator {
   CryptoppDigestCreator();
   virtual ~CryptoppDigestCreator();
 
-  size_t digestLength() override;
-  bool compute(const char* input,
-               size_t inputLength,
-               char* outBufferForDigest,
-               size_t lengthOfBufferForDigest) override;
-  void init() override {}
-  void update(const char* data, size_t len) override;
-  void finish(char* outDigest) override {}
-  void writeDigest(char* outDigest);
+  void updateDigest(const char* data, const size_t len) override;
```

Comment:
remove the word digest from all the function names, we can carry the digestLength name as its older name, but lets not add the word digest as its implicit.